### PR TITLE
Toast: "message" option runtime change should change visible toast's text (T1169347)

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

**Review by alexanderPolosatov — APPROVED** [review]

File: js/ui/toast/ui.toast.js [hunk 1]
```diff
@@ -176,12 +176,10 @@ const Toast = Overlay.inherit({
     },
 
     _renderContentImpl: function() {
-        if(this.option('message')) {
-            this._message = $('<div>')
-                .addClass(TOAST_MESSAGE_CLASS)
-                .text(this.option('message'))
-                .appendTo(this.$content());
-        }
+        this._message = $('<div>')
```
Comment (marker-dao) [inline js/ui/toast/ui.toast.js:179]: Why we should render message element here? Let's add method for it. And let's in optionChange method add follow code:

```
if(this._message) {
        this._message.text(args.value);
} else if (args.value) {
        this._yourRenderMessageMethod(args.value);
}
```

Comment (alexanderPolosatov) [reply js/ui/toast/ui.toast.js:179]: my opinion: Anton's way is more straight and easy to understand. this optimization (creating element only when message passed) do not give profit to us

Comment (marker-dao) [reply js/ui/toast/ui.toast.js:179]: Why?) This is an empty element. Browser spends resources on this. But it is not necessary. If we render such elements everywhere and every time, we will have a very slow codebase.

Comment (alexanderPolosatov) [reply js/ui/toast/ui.toast.js:179]: Did you see site with many toasts?)

Comment (ksercs) [reply js/ui/toast/ui.toast.js:179]: I think that there should be the single place where we render markup, and option change should only change the text

**Pull request merged**